scala/build/maintenance.Dockerfile

```
@@ -256,6 +256,13 @@ ENV PATH /home/qa/.local/bin:/home/qa/sbt/bin:/home/qa/bin:${PATH}
 RUN pipx install awscli
 # RUN ln -s /home/qa/.local/bin/aws /usr/local/bin/aws
 
+# trcli
+RUN apt-get update && apt-get install -y python3-full python3-pip
+RUN pip install --upgrade pip
+RUN pip install --user pipx
+RUN python3 -m pipx ensurepath
+RUN pipx install trcli
```

Hadolint

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)